### ISSUE TITLE
HomeKit port

### ISSUE THREAD

**Issue by farmio** [home-assistant/home-assistant.github.io#17247]
### Feedback

When setting up HomeKit via UI the port seems not to be configurable and in my case was not the default port stated in the documentation.
> TCP: 51827 (or the configured/used port in the integration settings).

A note could be added that the port is appended to the Bridge name.

### URL

https://www.home-assistant.io/integrations/homekit/

### Version

2021.3.4

**Comment by frenck** [comment]
> When setting up HomeKit via UI the port seems not to be configurable and in my case was not the default port stated in the documentation.

When adding multiple instances of HomeKit, different ports will be chosen (to prevent collision). This behavior is changed for 2021.4 a bit, and documented in the upcoming documentation.

> A note could be added that the port is appended to the Bridge name.

We don't document that. You can change the name (of any integration instance) to your liking using the rename button.

![image](https://user-images.githubusercontent.com/195327/113513114-50394500-9568-11eb-8f2a-d182c36b6a0a.png)

**Comment by farmio** [comment]
Then it would probably make sense to display the port in the options or somewhere. Having to activate debug logging to find out which port is used is not really user friendly. 
(I have not had multiple HomeKit instances).

**Comment by frenck** [comment]
> Then it would probably make sense to display the port in the options or somewhere. 

That is only available when configuring the integration manually, as documented.

**Comment by farmio** [comment]
Im not talking about setting it, but about displaying the automatically chosen port. Read only. 
I had problems in current beta with HomeKit. So I removed my yaml config and then, after restart, set up a new HomeKit integration via UI. This didn't work out of the box because it had chosen a different port than the previous yaml configured integration - which I had to add to my firewall config. 
I thought the number in the name was just a random identifier or had something to do with adding the bridge to HomeKit. Nowhere in the docs I found a note that this was the used port.
